app/services/page_flow_service.rb

```diff
@@ -0,0 +1,90 @@
+class PageFlowService
+  STEPS_FLOW = {
+    own_homes: {
+      path: :citizens_own_home_path,
+      back: nil,
+      forward: ->(application) { application.own_home_no? ? :savings_and_investments : :property_values }
+    },
+    property_values: {
+      path: :citizens_property_value_path,
+      back: :own_homes,
+      forward: ->(application) { application.own_home_mortgage? ? :outstanding_mortgages : :shared_ownerships }
+    },
+    outstanding_mortgages: {
+      path: :citizens_outstanding_mortgage_path,
+      back: :property_values,
+      forward: :shared_ownerships
+    },
+    shared_ownerships: {
+      path: :citizens_shared_ownership_path,
+      back: ->(application) { application.own_home_mortgage? ? :outstanding_mortgages : :property_values },
+      forward: ->(application) { application.shared_ownership? ? :percentage_homes : :savings_and_investments }
+    },
+    percentage_homes: {
+      path: :citizens_percentage_home_path,
+      back: :shared_ownerships,
+      forward: :savings_and_investments
+    },
+    savings_and_investments: {
+      path: :citizens_savings_and_investment_path,
+      back: lambda do |application|
```

Comment:
I think you can:
```ruby
back: -> do |application|
```
For consistency sake.

Reply:
By default, Rubocop wants `lambda do |x|` for multiline lambdas.
(https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/Lambda)
But I agree with you and I'm changing our Rubocop rules to always use `->(x)` instead of `lambda do |x|`